actions/Sorting.php

```diff
@@ -20,14 +20,15 @@ class Sorting extends Action
     public function run()
     {
         $transaction = \Yii::$app->db->beginTransaction();
+        $offset = \Yii::$app->request->post('offset');
```

Comment:
Hmmm... better is have a param for customization this. for example, I use another param name for offset and this can be not in POST :)